### PR TITLE
0.4.x: Deprecate "classic" template for removal in 1.0.0

### PR DESCRIPTION
**Pull request by martinRenou**
And allow for an alternative look which is `--template lab --show-margins`

### CODE REVIEW

File: voila/app.py [hunk 1]
```diff
@@ -83,7 +83,13 @@ class Voila(Application):
             },
             _("Set the log level to logging.DEBUG, and show exception tracebacks in output.")
         ),
-        'no-browser': ({'Voila': {'open_browser': False}}, _('Don\'t open the notebook in a browser after startup.'))
+        'no-browser': ({'Voila': {'open_browser': False}}, _('Don\'t open the notebook in a browser after startup.')),
+        'show-margins': (
+            {
+                'VoilaConfiguration': {'show_margins': True},
+            },
+            _('show left and right margins for the "lab" template, this gives a "classic" template look')
```
Comment (trungleduc) [inline voila/app.py:91]:
```suggestion
            _('Show left and right margins for the "lab" template, this gives a "classic" template look')
```

File: voila/configuration.py [hunk 2]
```diff
@@ -37,6 +37,13 @@ class VoilaConfiguration(traitlets.config.Configurable):
         """
     )
     theme = Unicode('light', config=True)
+    show_margins = Bool(
+        False,
+        config=True,
+        help=(
+            'show left and right margins for the "lab" template, this gives a "classic" template look'
```
Comment (trungleduc) [inline voila/configuration.py:44]:
```suggestion
            'Show left and right margins for the "lab" template, this gives a "classic" template look'
```

**Review by trungleduc — APPROVED** [review]

**Comment by jtpio** [comment]
Looks good, thanks!

**Comment by martinRenou** [comment]
Thank you both!

**Comment by martinRenou** [comment]
@meeseeksdev please backport to main

**Pull request merged**